#!/usr/bin/python3
def raise_exception_msg(message=""):
    raise NameError(message)

# Function that raises a name exception with a message.
# message = "C is Fun" because it was passed in from the main function
